frontend/components/DiskSpaceGraph/DiskSpaceGraph.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+
+import ReactTooltip from "react-tooltip";
+
+const renderDiskSpaceGraph = (
```

Comment:
Thanks for jumping in @ghernandez345 I basically followed the pattern from LastUpdatedText.tsx. I agree it would be more clear to use a component not a function, should I fix that for LastUpdatedText.tsx as well?

Reply:
I agree with Gabe's comment that the `<DiskSpaceGraph />` is more idiomatic React and we should probably adopt that syntax here. But FWIW I find the style that the React team chose for components is not particularly intuitive or readable compared to a well-named, well-typed function (like this one). I'm sure they had their reasons at the time as class-based components were probably kind of ugly to instantiate. But as it stands today, a React functional component is basically just a function with its own esoteric syntax that returns some JSX. There's some signaling value in the angle brackets I suppose but I'm not sure the  extra verbosity for the prop declarations and the overall cognitive shift away from normal function syntax is easier to reason about.

Reply:
i can see arguments for both being fine choices for how to render Components. The biggest value I think we gain from changing to exporting a traditional react component is that it follows the same pattern in the majority of our codebase.

Reply:
Totally agree!